source/_patterns/02-organisms/00-global/01-footer.mustache

```diff
@@ -1,83 +1,241 @@
 <footer class="siteFooter">
 	<nav class="siteFooter-nav small-footer hide-for-large-up">
-		<ul class="siteFooter-links">
-			<li class="footer-link-borrow">
-				<a href="{{borrow_href}}" kvtrackevent="Footer|click-Borrow-Apply now">Borrow</a>
-			<li class="footer-link-about">
-				<a href="{{about_href}}" kvtrackevent="Footer|click-Get to know us-About us">About</a>
-			<li class="footer-link-help">
-				<a href="{{help_href}}" kvtrackevent="Footer|click-Get to know us-Help">Help</a>
-			<li class="footer-link-careers">
-				<a href="{{careers_href}}" target="_blank" kvtrackevent="Footer|click-Work with us-Careers">Careers</a>
-			<li class="footer-link-map"><a href="{{site_map_href}}" kvtrackevent="Footer|click-Explore-Site map">Site map</a>
-		</ul>
-
-		<div class="siteFooter-policy">
-			<a href="{{privacy_policy_href}}" class="siteFooter-privacy" kvtrackevent="Footer|click-Explore-Privacy policy">Privacy policy</a>
-			<a href="{{cookie_policy_href}}" class="siteFooter-cookie" kvtrackevent="Footer|click-Explore-Cookie policy">Cookie and Data Settings</a>
-			<a href="{{tou_href}}" class="siteFooter-tos" kvtrackevent="Footer|click-Explore-Terms of use">Terms of use</a>
+		<div class="siteFooter-borrow-section siteFooter-group">
+			<h2>
+				Borrow
+			</h2>
+			<p>
+				Loans for entrepreneurs doing amazing things.
+			</p>
+			<ul class="siteFooter-links">
+				<li>
+					<a href="{{borrow_href}}" class="borrow_href" kvtrackevent="Footer|click-Borrow-Apply now">Apply now</a>
+				</li>
+				{{#request_url_borrow}}
+				<li class="footer-link-apply-language-switch">
+					English&nbsp;
+					<a href="{{borrower_landing_spanish_href}}" class="borrower_landing_spanish_href" kvtrackevent="Footer|click-apply-langugage-link|Español">Español</a>
+				</li>
+				{{/request_url_borrow}}
+			</ul>
 		</div>
 
-        {{#request_url_borrow}}
-        <div class="siteFooter-policy">
-            English&nbsp;
-            <a href="{{borrower_landing_spanish_href}}" class="borrower_landing_spanish_href">Español</a>
-        </div>
-        {{/request_url_borrow}}
-
-		<div class="siteFooter-group">
-			<ul class="download-app-icons">
+		<div class="siteFooter-lend-section siteFooter-group">
+			<h2>
+				Lend
+			</h2>
+			<p>
+				Make a loan, change a life.
+			</p>
+			<ul class="siteFooter-links">
 				<li>
-					<a
-						href="{{app-store_href}}"
-						target="_blank" rel="noopener"
-						kvtrackevent="global|click-app-badge-footer|App Store"
-					>
-						<img
-							class="download-app-ios"
-							src="//images.ctfassets.net/j0p9a6ql0rn7/3ZDltzdfhQHRwpRvnlrq6v/b97cfd1f1354e0276861a7aca110d738/app-store.svg"
-							loading="lazy"
-							alt="Check out our new app on the App Store"
-						>
-					</a>
+					<a href="{{lend_by_category_href}}" class="lbc_href" kvtrackevent="Footer|click-Lend-lend now">Lend now</a>
```

Comment:
Should these links still open in new tabs?

Reply:
Generally, these internal links open in the same tab/window and that is this is based on the Ui mobile footer. The App links do open in a new tab/window though. They are further down in these code additions.